phoenix-hive/src/test/java/org/apache/phoenix/hive/query/PhoenixQueryBuilderTest.java

```diff
@@ -170,4 +175,67 @@ public void testBuildQueryWithNotNull() throws IOException {
                         "\"" + COLUMN_DATE + "\" is not null ",
                 BUILDER.buildQuery(jobConf, TABLE_NAME, readColumnList, searchConditions));
     }
+
+    @Test
+    public void testBuildQueryWithBigintColumns() throws IOException {
+        final String COLUMN_BIGINT = "Column_Bigint";
+        final String COLUMN_INT = "Column_Int";
+        final String COLUMN_BIGINT2 = "Column_Bigint2";
+        final String tableName = "TEST_TABLE";
+        final String expectedQueryPrefix = "select /*+ NO_CACHE  */ * from " + tableName
+          + " where ";
+
+        JobConf jobConf = new JobConf();
+        List<String> readColumnList = Collections.emptyList();
+
+        {
```

Comment:
Can you get rid of the curly-braces? I think it's fine to have these three clauses in this one test method, but, if you'd like to have them more logically separated, separate test methods would also be fine.